dashboard/app/views/devise/shared/_oauth_links.haml

```diff
@@ -32,3 +32,10 @@
   if (!!window.MakerBridge) {
     $('.with-google_oauth2').prop({'disabled': true, 'title': 'Please use the ‘Log in with Google’ button in the toolbar at the top of the Maker App. If you do not see a ‘Log in with Google’ button, please download the latest Maker App version.'});
   }
+
+  var facebookButton = document.getElementsByClassName('oauth-sign-in with-facebook')[0];
```

Comment:
Selected by `class` here in order to keep this change lightweight. It will eventually be reverted so didn't want to make any updates to the existing code if possible.